minard/templates/nhit_monitor_thresholds_nearline.html

```diff
@@ -27,10 +27,10 @@
 		<table class="table">
 		    <thead>
                         <tr>
-                            <th></th>
+                            <th> Run range: <input style=width:100px;" type="text" id="low" value={{run_range_low}} onKeyDown="if(event.keyCode==13) sorting_by({{limit}}, {{offset}}, 'run', this.value, high.value);"> </input> - <input type="text" style=width:100px;" id="high" value={{run_range_high}} onKeyDown="if(event.keyCode==13) sorting_by({{limit}}, {{offset}}, 'run', low.value, this.value);"></input> </th>
```

Comment:
This shouldn't be in the table as it causes the key column to be too big. I think it should also be a `<input type="number">` and there shouldn't be any need to specify the pixel width (should change based on the browser width).